index.bs

```diff
@@ -1509,6 +1513,10 @@ during registration.
         [=attestation statement=], as well as to decode and validate the [=authenticator data=] along with the
         [=JSON-serialized client data=]. For more details, see [[#sctn-attestation]], [[#generating-an-attestation-object]],
         and [Figure 3](#fig-attStructs).
+
+    :   <dfn>transports</dfn>
+    ::  This attribute indicates which [transports](#transport) the [=client=] believes can be used to communicate with
+        this [=authenticator=].
```

Comment:
How does this interact with anonymization, like the `none` attestation? If the user somehow requests maximum privacy, is it acceptable for `transports` to be the empty set? (I'm thinking Tor Browser here, not mainline Firefox.)